src/api/routes/applications/index.ts

```diff
@@ -68,6 +69,26 @@ router.post(
 			flags: 0,
 		});
 
+		// april 14, 2023: discord made bot users be automatically added to all new apps
+		const { autoCreateBotUsers } = Config.get().general;
+		if (autoCreateBotUsers) {
+			const user = await User.register({
+				username: app.name,
+				password: undefined,
+				id: app.id,
+				req,
+			});
+
+			user.id = app.id;
+			user.premium_since = new Date();
+			user.bot = true;
+
+			await user.save();
+
+			// flags is NaN here?
+			app.assign({ bot: user, flags: app.flags || 0 });
+		}
```

Comment:
this is just copy pasted from [here](https://github.com/spacebarchat/server/blob/master/src/api/routes/applications/%23id/bot/index.ts#L55-L69) right? wouldn't it be better to move this isn't a separate function so we don't have to remember to edit in 2 places?

Reply:
yeah, I wondered about that. I'll add the change